### ISSUE TITLE
[Feature Request]: Use hash to encrypt rpc-secret.

### ISSUE THREAD

**Issue by SkyYkb** [aria2/aria2#1678]
When use aria2c as a rpc server, usually we need to save `rpc-secret` in a configuration file.
But we can only save `rpc-secret` as plain text. That's not secure.
So I would suggest to use md5 or something to encrypt `rpc-secret`.

Thanks.

**Comment by levicki** [comment]
@SkyYkb Not even sha1 is secure, let alone md5. Adding hashing or encryption would not improve security. Instead, you should make sure unauthorized users cannot access and read your `aria2.conf` file.

**Comment by SkyYkb** [comment]
Yeah, I certainly know I should prevent unauthorized access to the conf file, but however, I would never like to see that my password was saved as plain text.
Other programs also provided other encryption algorithms like encrypting with tool htpasswd. Md5 is only as an example.

**Comment by binux** [comment]
How about just hash your password and use it as rpc-secret.

**Comment by SkyYkb** [comment]
But it’s hard to memorize a hash string...
And however the password saved on the server is still plain text…

**Comment by levicki** [comment]
If someone has access to your server you have a much bigger problem to worry about than them getting to your RPC secret.
